.chloggen/add tls tests to hec receiver and exporter.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: splunkhecreceiver, splunkhecexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: add tls tests to hec receiver and exporter
```

Comment:
no need for a changelog if no functional changes are present.

Reply:
thank you. Just notice the tag.

Reply:
No problem. You can also have your PR title start with `[chore]` to skip changelog.